streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockTrackerSuite.scala

```diff
@@ -225,7 +225,7 @@ class ReceivedBlockTrackerSuite
   /** Generate blocks infos using random ids */
   def generateBlockInfos(): Seq[ReceivedBlockInfo] = {
     List.fill(5)(ReceivedBlockInfo(streamId, Some(0L), None,
-      BlockManagerBasedStoreResult(StreamBlockId(streamId, math.abs(Random.nextInt)))))
+      BlockManagerBasedStoreResult(StreamBlockId(streamId, math.abs(Random.nextInt)), Some(0L))))
```

Comment:
I did not change the Random.nextInt. Just added 0 as BlockManagerBasedStoreResult now takes the recordCount. This testcase is for testing the recordcount 0 which @zsxwing did, which I changed after the merge.